### PR TITLE
[suggestion] added plain text format#4218

### LINKED ISSUE #4218: [suggestion] Remove JSON encoding for API Version and Health requests, use plain text
## Description

When you `GET /api_version`, you get a JSON:

```json
"1"
```

I think it is redundant to use JSON here and plain text would be just fine:

```
1
```

Same applies for `GET /health`:

```json
"Healthy"
```

There is no any reason to use JSON here over plain text.

### Benefits

Simplify API a little bit, remove redundancy.

### Research

A simple patch of this sections would suffice:

https://github.com/hyperledger/iroha/blob/565271dd043b6371c1fc2e4edce0754a3058c1bb/torii/src/routing.rs#L308-L317

https://github.com/hyperledger/iroha/blob/565271dd043b6371c1fc2e4edce0754a3058c1bb/torii/src/routing.rs#L139-L141

### PR DESCRIPTION
**Pull request by gerceboss**
### Linked issue

Closes #4218 

### Benefits
removes redundancy

### Checklist

- [x] I've read `CONTRIBUTING.md`
- [x] I've used the standard signed-off commit format (or will squash just before merging)
- [x] All applicable CI checks pass (or I promised to make them pass later)
- [ ] (optional) I've written unit tests for the code changes
- [x] I replied to all comments after code review, marking all implemented changes with thumbs up

### CODE REVIEW

File: torii/src/routing.rs [hunk 1]
```diff
@@ -136,8 +136,8 @@ enum Health {
     Healthy,
 }
 
-pub fn handle_health() -> Json {
-    reply::json(&Health::Healthy)
+pub fn handle_health() -> impl warp::Reply {
+    warp::reply::with_status("Healthy", StatusCode::OK).header("Content-Type", "text/plain")
```
Comment (DCNick3) [inline torii/src/routing.rs:140]: Returning a `&'static str` will be more terse and will still correctly set the `Content-Type` header: https://docs.rs/warp/latest/src/warp/reply.rs.html#436-443

File: torii/src/routing.rs [hunk 2]
```diff
@@ -305,15 +305,15 @@ pub mod subscription {
...
+pub async fn handle_version(sumeragi: SumeragiHandle) -> impl warp::Reply  {
     use iroha_version::Version;
 
     let string = sumeragi
         .apply_wsv(WorldStateView::latest_block_ref)
         .expect("Genesis not applied. Nothing we can do. Solve the issue and rerun.")
         .version()
         .to_string();
-    reply::json(&string)
+    warp::reply::with_status(string, warp::http::StatusCode::OK).header("Content-Type", "text/plain")
```
Comment (DCNick3) [inline torii/src/routing.rs:316]: Same, but can return the version `String` directly

**Comment by DCNick3** [comment]
Also, please make sure your commit messages follow the [format we maintain](https://github.com/hyperledger/iroha/blob/iroha2-dev/CONTRIBUTING.md#git-workflow). This PR also should be a single commit (not four), as it is a small change

**Comment by gerceboss** [comment]
No problem ,I will do that by tomorrow

**Comment by mversic** [comment]
@gerceboss will you finalize this PR?

**Comment by 6r1d** [comment]
@gerceboss please comment :-)

**Pull request closed without merging**